traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -317,6 +323,9 @@ func LoadConfig(cdnConfPath string, dbConfPath string, appVersion string) (Confi
 	if err != nil {
 		return Config{}, []error{fmt.Errorf("unmarshalling '%s': %v", dbConfPath, err)}, BlockStartup
 	}
+	if portNum, err := strconv.Atoi(cfg.DB.Port); err != nil || portNum < MinPort || MaxPort < portNum {
+		cfg.DB.Port = DefaultDBPort
```

Comment:
Do you think we should print an error here, saying that the port in the config file was invalid, so we're defaulting to 5432?

Reply:
done